UnitsNet.Serialization.JsonNet/UnitsNetJsonConverter.cs

```diff
@@ -93,7 +93,10 @@ public class UnitsNetJsonConverter : JsonConverter
             object unit = Enum.Parse(reflectedUnitEnumType, unitEnumValue);
 
             // Mass.From() method, assume no overloads exist
-            MethodInfo fromMethod = reflectedUnitType.GetMethod("From");
+            MethodInfo fromMethod = (from m in reflectedUnitType.GetMethods()
+                                     where m.Name.Equals("From", StringComparison.InvariantCulture) &&
+                                     !m.ReturnType.IsGenericType  // we want the non nullable type
```

Comment:
I just noticed this. It probably works already, but I found a post on how to properly test for nullable.

http://stackoverflow.com/a/374663/134761

Do you think we should use that method instead?